docs/ingestion/native-batch.md

```diff
@@ -1310,6 +1311,43 @@ A spec that applies a filter and reads a subset of the original datasource's col
 This spec above will only return the `page`, `user` dimensions and `added` metric.
 Only rows where `page` = `Druid` will be returned.
 
+### Sql Input Source
```

Comment:
One more thing, I remember that many people from our community have been asking about how to use `SqlFirehose`. What do you think about adding a section that explains how to use it in production environment? To be honest, it's not clear for me what are best practices to make a scalable and efficient pipeline using this input source. For example, how do you parallelize each ingestion task (which means, how do you split queries)? How do you handle data updates in database after ingestion? How often do you run ingestion jobs? and so on.

Reply:
@jihoonson I have added more details related to this in the docs. Please take a look. Since this is a one time ingestion from rdbms, updates to the source db data for a specific interval would require a new sql ingestion task be spawned for the same interval which will replace the old segments.

By the way, I would like to add a continuous ingestion from RDBMS feature as well which could use the indexing service with a RDBMS supervisor. Do you think this would be a useful feature?

Reply:
I would definitely vote for having such a supervisor! That will be super useful.